app/models/standup_meeting.rb

```diff
@@ -42,4 +44,19 @@ class StandupMeeting < ApplicationRecord
   }
 
   scope :for_member, ->(user, group) { where(user:, standup_meeting_group: group) }
+
+  def comments(section)
+    standup_meeting_comments.where(name: section)
+  end
+
+  def section_content(section)
+    case section
+    when 'yesterday_work_description'
+      yesterday_work_description
+    when 'today_work_description'
+      today_work_description
+    when 'blockers_description'
+      blockers_description
+    end
+  end
```

Comment:
Used in sections index view to query the dynamically render the appropriate comments